pkg/gadgets/socket-collector/gadget.go

```diff
@@ -51,7 +51,7 @@ func (f *TraceFactory) Operations() map[string]gadgets.TraceOperation {
 
 	return map[string]gadgets.TraceOperation{
 		"start": {
-			Doc: "Collect a snapshot of the list of sockets",
+			Doc: "Create a snapshot of the active TCP and UDP sockets",
```

Comment:
It also show so-called passive TCP sockets (TCP sockets in the LISTEN state)
```suggestion
			Doc: "Create a snapshot of the TCP and UDP sockets",
```

Reply:
Yeah, ok. Active was the wrong word and I knew this, but I wanted to highlight that they were the current ones. But "current sockets" also sounds wrong. :-/ Would "currently open sockets" be the right expression?

Reply:
I thought that "snapshot" would convey that. What about:

> "Take a snapshot of the currently open TCP and UDP sockets. Once taken, the snapshot is not updated automatically. However one can call the `start` operation again at any time to update the snapshot."

Reply:
I didn't know this could be a longer string.  Alright, I've taken that and added the same sentence to the process one. PTAL?

Reply:
LGTM!